Sources/PointFree/TODO.swift

```diff
@@ -61,19 +61,6 @@ public func requireSome<A>(
     }
 }
 
-extension PartialIso where A == String, B == String {
-  public static func decrypted(withSecret secret: String) -> PartialIso<String, String> {
-    return PartialIso(
-      apply: { HttpPipeline.decrypted(text: $0, secret: secret) },
-      unapply: { encrypted(text: $0, secret: secret) }
-    )
-  }
-
-  public static var appDecrypted: PartialIso<String, String> {
```

Comment:
Was going to convert this to use `Encrypted<String>`, but we weren't using this.